web_widget_color/__openerp__.py

```diff
@@ -39,6 +39,6 @@
         'static/src/xml/widget.xml',
```

Comment:
Hi @llacroix  thanks for your contribution
I will check mostly format and guidelines in your PR.
* Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
* Adapt the version number: 9.0.1.0.0